certify/dns_certify.mli

```diff
@@ -13,6 +13,30 @@ val letsencrypt_name : 'a Domain_name.t ->
 (** [letsencrypt_name host] is the service name at which we store let's encrypt
     certificates for the [host]. *)
 
+val is_csr : Dns.Tlsa.t -> bool
+(** [is_csr tlsa] is true if [tlsa] is a certificate signing request of
+    interest for this library. *)
```

Comment:
What does "of interest for this library" mean here?

Reply:
"this library" is dns-certify, which uses TLSA records for certificates.

Reply:
Right, but what makes a certificate signing request interesting/not interesting to the library?